Too high volume brings clipping.
Thanks Amie Amaman <amie303@yahoo.de> for the investigation

--- a/pico/lib/picosig.c
+++ b/pico/lib/picosig.c
@@ -173,7 +173,7 @@ static pico_status_t sigInitialize(regis
          * ------------------------------------------------------------------*/
         /*pitch , volume , speaker modifiers*/
         sig_subObj->pMod = 1.0f;
-        sig_subObj->vMod = 1.0f;
+        sig_subObj->vMod = 0.5f;
         sig_subObj->sMod = 1.0f;
     } else {
         /*-----------------------------------------------------------------
--- a/pico/lib/picotok.c
+++ b/pico/lib/picotok.c
@@ -757,7 +757,7 @@ static void tok_interpretMarkup (picodat
                     tok_putItem(this, tok, PICODATA_ITEM_CMD, PICODATA_ITEMINFO1_CMD_SPEAKER, PICODATA_ITEMINFO2_CMD_RELATIVE, uval, (picoos_uchar*)"");
                 } else {
                     uval = picoos_atoi((picoos_char*)tok->markupParams[0].paramVal);
-                    tok_checkLimits(this, & uval, PICO_SPEAKER_MIN, PICO_SPEAKER_MAX, (picoos_uchar*)"volume");
+                    tok_checkLimits(this, & uval, PICO_SPEAKER_MIN, PICO_SPEAKER_MAX, (picoos_uchar*)"speaker");
                     tok_putItem(this, tok, PICODATA_ITEM_CMD, PICODATA_ITEMINFO1_CMD_SPEAKER, PICODATA_ITEMINFO2_CMD_ABSOLUTE, uval, (picoos_uchar*)"");
                 }
                 done = TRUE;
